shared_model/bindings/CMakeLists.txt

```diff
@@ -46,9 +46,9 @@ if (SWIG_PYTHON)
         set(MAC_OPTS "-flat_namespace -undefined suppress")
     endif()
 
-    swig_add_library(irohapy LANGUAGE python SOURCES bindings.i)
-    swig_link_libraries(irohapy ${Python_LIBRARIES} bindings ${MAC_OPTS})
-    add_custom_target(irohapy DEPENDS ${SWIG_MODULE_irohapy_REAL_NAME})
+    swig_add_library(iroha LANGUAGE python SOURCES bindings.i)
```

Comment:
why iroha, but not irohapy?

Reply:
Because of lovely Swig. For Python and only for Python it takes module name from %module directive from interface file (.i) so it will be iroha regardless you write here. Even worse - if you write here "irohapy" in swig_add_library it will not work it all because internally it uses plain "iroha" everywhere.